scripts/lcow/Run-hypervtest.ps1

```diff
@@ -61,6 +59,32 @@ function Main {
     Copy-Item $WorkDir\log*\* -Destination $LogDestination -Recurse
 
     Pop-Location
+
+    $summaryPath = Join-Path $LogDestination "AppSummary.csv"
+    if (Test-Path $summaryPath) {
+        $summaryContent = Get-Content $summaryPath -Raw
+    } else {
+        throw "Cannot find summary file: AppSummary.csv"
+    }
+
+    $failureNumber = 0
+
+    $testResults = $($summaryContent -split "`r`n`r`n")
+    foreach ($result in $testResults) {
+        if ($result -eq "" ) {
+            continue
+        }
+        $result =  $result.Split("`n")
+        if ($result[12].Split(":")[1].Trim() -ne "0") {
+            $failureNumber ++
+        }
+    }
+    if ($failureNumber -ge $BaseNumber) {
+        Write-Host "Tests fail more that expected"
```

Comment:
Expected $value number of failures, total number of failures for this run was $value